pkg/testcase/cluster.go

```diff
@@ -6,44 +6,102 @@ import (
 
 	"github.com/rancher/distros-test-framework/factory"
 	"github.com/rancher/distros-test-framework/shared"
+	"github.com/rancher/distros-test-framework/pkg/customflag"
+
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 )
 
+var arch = customflag.ServiceFlag.ClusterConfig.Arch.String()
```

Comment:
this var here it would not work....bc this should be instatiate on the test main. did you tested it ?